### DISCUSSION TITLE
How to control the interval between of sending net package?

### DISCUSSION THREAD

**Discussion by Squirre17** [jtpereyda/boofuzz#659] [Q&A] [answered]
I found that the interval of two package transmissions is too short, exceeding the target server's handling ability. What can I do if I want to slow down the sending speed?

**Comment by SR4ven** [comment, marked as answer]
Hi @Squirre17.
There is a `Session` parameter called `sleep_time` to wait for a given a given time before proceeding with the next test case.
https://boofuzz.readthedocs.io/en/stable/source/Session.html
